onnx/hub.py

```diff
@@ -363,12 +363,35 @@ def download_model_with_test_data(
                 "download the model from the model hub."
             )
 
+    def members_filter(tarfile, base):
```

Comment:
Please move the file to the module namespace. Add type annotations, and rename to `_tar_members_filter`.

Reply:
moved and renamed to `_tar_members_filter`